pom.xml

```diff
@@ -173,12 +186,41 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-
+            <dependency>
+                <groupId>redis.clients.authentication</groupId>
+                <artifactId>redis-authx-core</artifactId>
+                <version>0.1.0-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>redis.clients.authentication</groupId>
+                <artifactId>redis-authx-entraid</artifactId>
+                <version>0.1.0-SNAPSHOT</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>io.github.cdimascio</groupId>
+                <artifactId>dotenv-java</artifactId>
+                <version>2.2.0</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
IMHO we do not need these

Reply:
Ok. Wil apply the change when switching to non SNAPSHOT versiosn

Reply:
Removed